src/Command/SynchronizeProductsCommand.php

```diff
@@ -50,17 +50,19 @@ public function __construct(
         $this->productRepository = $productRepository;
     }
 
-    protected function configure()
+    protected function configure():void
     {
         $this->setName('sulu-sylius:synchronize:products')
             ->setDescription('Synchronize all products to Sulu');
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output):int
     {
         parent::execute($input, $output);
 
         $this->syncProducts($output);
+
+        return 1;
```

Comment:
this indicates that an error happened, right?

Reply:
🙈 your right